Add --strict option to custom bundle (#6557)
plotly/plotly.js: draftlogs/6557_add.md

> - Add --strict option to custom bundle command so that strict traces can be used in a custom bundle. [[#6557](https://github.com/plotly/plotly.js/pull/6557)]

Reviewer: ```suggestion
 - Add strict option to custom bundle command [[#6557](https://github.com/plotly/plotly.js/pull/6557)], with thanks to @CallumNZ for the contribution!
 ```
